app/controllers/supplementary/SummaryPageController.scala

```diff
@@ -49,13 +51,18 @@ class SummaryPageController @Inject()(
 )(implicit ec: ExecutionContext)
     extends FrontendController with I18nSupport {
 
+  implicit val appConfigImpl: AppConfig = appConfig
```

Comment:
I mean to make this app conf implcit like ec

`(implicit ec: ExecutionContext, appConfig: AppConfig)`

Reply:
And remove from 42 line